### ISSUE TITLE
Android View loosing styles

### ISSUE THREAD

**Issue by gsdias** [GeekyAnts/NativeBase#571]
"native-base": "^2.0.8",
    "react": "15.4.2",
    "react-native": "0.41.2",

When using View from nativa-base and combining the styles borderRadius and borderWidth/borderColor the border color/width is not rendered, android only.

`<View style={{ borderRadius: 10, borderWidth: 1, borderColor: 'white' }} />`

If I use View from react-native works fine.

**Comment by davidbq** [comment]
Same problem here. 

In my case I am using an `Input` inside an `Item` and I'm setting a custom style:

`errorItemStyle: {
        flex: 1,
        margin: 5,
        borderRadius: 5,
        backgroundColor: '#ffeeee',
        height: 40,
        borderWidth: 1,
        borderColor: '#ff0a13'
      }`

 On iOS is working fine, but on Android the border is not shown.

I have also tried to set `borderTopWidth`, `borderRightWidth` ... But still doesn't work.

Any suggestion? Thanks in advance.

**Comment by samanthacampisi** [comment]
Yes, I'm facing the same problem too. Any workarounds?

**Comment by adityasaxena** [comment]
@davidbq @samanthacampisi @gsdias Hello! we've played around with NativeBase as well. And, we have suffered the same issues

**Comment by breogangf** [comment]
@davidbq @samanthacampisi @adityasaxena We are experiencing the same issues here!

**Comment by sanketsahusoft** [comment]
@gsdias @davidbq @samanthacampisi @adityasaxena @breogangf We are looking into it. Hope to have a working fix today.

**Comment by breogangf** [comment]
Thank you @sanketsahusoft, if you need me to test anything or help, just ask ;-)

**Comment by sankhadeeproy007** [comment]
Can anybody check if it works with 2.0.11?

**Comment by davidbq** [comment]
It is working now with version 2.0.11!!!!! Thanks a lot!!!

**Comment by davidbq** [comment]
Actually, I've spoken too fast. It is working for Android now, but it has a weird behavior on iOS, It only shows the bottom border, this is what I get:
![screen shot 2017-03-01 at 11 03 02](https://cloud.githubusercontent.com/assets/6732229/23455241/2b92a67c-fe6f-11e6-986d-eec302399c2b.png)
And this is the Android one which is working perfectly:
![screen shot 2017-03-01 at 11 04 31](https://cloud.githubusercontent.com/assets/6732229/23455276/4518790a-fe6f-11e6-85a6-aac46139e901.png)

**Comment by sankhadeeproy007** [comment]
Can you confirm that this is the case just for input? Is it similar to #582?

**Comment by davidbq** [comment]
I have just tried to set a border to a button and an image and it works fine for both Android and iOS.

Regarding to #582, in my case I am not using `<Form>`, I have something like this: 
```
<View>
    <Item style={errorItemStyle}>
        <Input style={inputTextStyle}/>
    </Item>
</View>
```
```
errorItemStyle: {
    flex: 1,
    margin: 5,
    borderRadius: 5,
    backgroundColor: '#ffeeee',
    height: 40,
    borderWidth: 1,
    borderColor: '#ff0a13'
},
inputTextStyle: {
    paddingLeft: 10,
    paddingBottom: 10,
    paddingRight: 10
}
```

**Comment by sankhadeeproy007** [comment]
It's related to `Item` I guess then

**Comment by davidbq** [comment]
Yes, it seems like. That's why I added my "itemStyle" on my first post, because it is where I set the border.

**Comment by sankhadeeproy007** [comment]
Okay, will look into it.

**Comment by breogangf** [comment]
Do we have any news about this?

**Comment by davidbq** [comment]
I just realised this is rendering wrongly only on the emulator. On real devices is working for both Android and iOS

**Comment by shivrajkumar** [comment]
Fixed in v2.1.3
